### PR TITLE
feat: Raise exception in tests when blocking code is called in event loop

### PR DESCRIPTION
**Pull request by cbornet**
Raise exception in tests when blocking code is called in event loop

### CODE REVIEW

File: src/backend/tests/blockbuster.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        if self.getblocking():
+            msg = f"Blocking call to {func.__module__}.{func.__qualname__}"
+            raise BlockingError(msg)
+        return func(self, *args, **kwargs)
+
+    return socket_op
+
+
+def init():
+    time.sleep = _raise_if_blocking(time.sleep)
```
Comment (cbornet) [inline src/backend/tests/blockbuster.py:52]: One drawback of patching time.sleep is that debugging with breakpoints in test will raise the `BlockingError`.
So this behavior should probably be configurable.

File: src/backend/base/langflow/services/telemetry/service.py [hunk 2]
```diff
@@ -93,15 +94,16 @@ async def _queue_event(self, payload) -> None:
     async def log_package_version(self) -> None:
         python_version = ".".join(platform.python_version().split(".")[:2])
         version_info = get_version_info()
-        architecture = platform.architecture()[0]
```
Comment (cbornet) [inline src/backend/base/langflow/services/telemetry/service.py:96]: Detected by the block buster 😀

File: src/backend/tests/blockbuster.py (hunk 1 above)
Comment (ogabrielluiz) [reply src/backend/tests/blockbuster.py:52]: Maybe a fixture? I debuge a lot through tests

Comment (cbornet) [reply src/backend/tests/blockbuster.py:52]: Yes, me too.
Maybe there could be en env var `RAISE_ON_BLOCKING_TIME_SLEEP` that we set by default to `false`. And that we set to `true` in CI ?

Comment (ogabrielluiz) [reply src/backend/tests/blockbuster.py:52]: I like that option

Comment (cbornet) [reply src/backend/tests/blockbuster.py]: I found another way by inspecting the call stack.
I've checked with PyCharm and will also check with VSCode.

Comment (cbornet) [reply src/backend/tests/blockbuster.py]: It's OK with VSCode.

**Review by ogabrielluiz — APPROVED** [review]
This is awesome. 

LGTM

**Pull request merged**